chapter_12_dependency_injection.asciidoc

```diff
@@ -244,15 +258,21 @@ TODO: betterify diagram
 
 
 
-=== Preparing Handlers: Poor Man's DI with Partial Functions
+=== Preparing Handlers: Poor Man's DI with Closures and Partials
 
 One way to turn a function with dependencies into one that's ready to be
-called later with those dependencies _aleady injected_, is to use partial
-functions:
+called later with those dependencies _aleady injected_, is to use closures or
+partial functions to compose the function with its dependencies:
+
+//TODO (EJ1) Re the following examples, there is a difference
```

Comment:
Ed you are an absolute legend.  I had no idea.

Reply:
I made a small snippet to demonstrate, in case you want one. https://github.com/cosmicpython/book/issues/204

For this section I think it's just sufficent to mention that they're different.